src/js/components/Button/index.d.ts

```diff
@@ -11,7 +11,7 @@ import {
   Omit,
   PolymorphicType,
 } from '../../utils';
-
+import { ButtonKindType } from 'src/js/themes/base';
```

Comment:
Rationale for explicit path versus relative here (e.g. `'../../themes/base'`)?

Reply:
I think it auto-imported. let me update to relative.